Sources/Core/Shared/OperationQueue.swift

```diff
@@ -197,7 +197,7 @@ public extension OperationQueue {
 
      - returns: The main queue
      */
-    public override class func main() -> OperationQueue {
+    public class func main() -> OperationQueue {
```

Comment:
I was not sure what to do here. According to the documentation, your original line should be correct. However, the compiler claims that function does not exist in the superclass.

I figured if it has really been removed from the superclass, then only other operations code would call it and it is returning a variable made here anyway, so I thought this would be safe.